dolphinscheduler-task-plugin/dolphinscheduler-task-remoteshell/src/main/java/org/apache/dolphinscheduler/plugin/task/remoteshell/RemoteExecutor.java

```diff
@@ -156,7 +156,6 @@ public void kill(String taskId) throws IOException {
         String pid = getTaskPid(taskId);
         String killCommand = String.format(COMMAND.KILL_COMMAND, pid);
         runRemote(killCommand);
-        cleanData(taskId);
```

Comment:
Could you please explain more why we need to delete cleanData() here?

Reply:
@rickchengx Hi, I read the logic again, we can not delete `cleanData() `here. I think, The `rm` command in `cleanData()` should be replaced to `rm -f` to avoid the printing of the error log when the file is not existed. what you think?

Reply:
> @rickchengx Hi, I read the logic again, we can not delete `cleanData() `here. I think, The `rm` command in `cleanData()` should be replaced to `rm -f` to avoid the printing of the error log when the file is not existed. what you think?

I think we should make sure the remote task can be killed successfully. And There can be various reasons for errors when cleaning data, and I think it's acceptable to make users aware of these errors.

Reply:
> > @rickchengx Hi, I read the logic again, we can not delete `cleanData() `here. I think, The `rm` command in `cleanData()` should be replaced to `rm -f` to avoid the printing of the error log when the file is not existed. what you think?
> 
> I think we should make sure the remote task can be killed successfully. And There can be various reasons for errors when cleaning data, and I think it's acceptable to make users aware of these errors.

sure，I will revert to this logic later。

Reply:
> > @rickchengx Hi, I read the logic again, we can not delete `cleanData() `here. I think, The `rm` command in `cleanData()` should be replaced to `rm -f` to avoid the printing of the error log when the file is not existed. what you think?
> 
> I think we should make sure the remote task can be killed successfully. And There can be various reasons for errors when cleaning data, and I think it's acceptable to make users aware of these errors.

Hi,  I reverted `cleanData()` logic.